website/cue/reference/components/sources/dnstap.cue

```diff
@@ -1240,10 +1240,12 @@ components: sources: dnstap: {
 	}
 
 	telemetry: metrics: {
-		events_in_total:                 components.sources.internal_metrics.output.metrics.events_in_total
-		processed_bytes_total:           components.sources.internal_metrics.output.metrics.processed_bytes_total
-		processed_events_total:          components.sources.internal_metrics.output.metrics.processed_events_total
-		parse_errors_total:              components.sources.internal_metrics.output.metrics.parse_errors_total
-		component_received_events_total: components.sources.internal_metrics.output.metrics.component_received_events_total
+		events_in_total:                      components.sources.internal_metrics.output.metrics.events_in_total
+		processed_bytes_total:                components.sources.internal_metrics.output.metrics.processed_bytes_total
+		processed_events_total:               components.sources.internal_metrics.output.metrics.processed_events_total
+		parse_errors_total:                   components.sources.internal_metrics.output.metrics.parse_errors_total
+		component_received_bytes_total:       components.sources.internal_metrics.output.metrics.component_received_bytes_total
+		component_received_events_total:      components.sources.internal_metrics.output.metrics.component_received_events_total
+		component_received_event_bytes_total: components.sources.internal_metrics.output.metrics.component_received_event_bytes_total
```

Comment:
We can include the `component_sent_*` metrics here.

Reply:
They appear to be in [here](https://github.com/vectordotdev/vector/blob/master/website/cue/reference/components/sources.cue#L346) and they are added automagically to [every source](https://vector.dev/docs/reference/configuration/sources/dnstap/#component_sent_events_total).